test/unit/typography/font_loading.js

```diff
@@ -59,7 +59,7 @@ suite('Fonts outside preload()', function () {
 
     test('should call success-callback with object when font loads',
       function (done) {
-        p5Inst.loadFont('manual-test-examples/p5.Font/acmesa.ttf',
+        p5Inst.loadFont('../manual-test-examples/p5.Font/acmesa.ttf',
```

Comment:
we can do away with the `complete-tests-build` directory, and keep the HTML in the root of test/ like it was before